views/statisticsView.vue

```diff
@@ -0,0 +1,72 @@
+<script
+    lang="ts">
+import { UserModel } from "~/model/UserModel";
+import { reactive } from "vue";
+
+import { Line } from 'vue-chartjs'
+import { Chart as ChartJS, Title, Tooltip, Legend, PointElement, LineElement, CategoryScale, LinearScale } from 'chart.js'
+
+ChartJS.register(CategoryScale,
+    LinearScale,
+    PointElement,
+    LineElement,
+    Title,
+    Tooltip,
+    Legend)
+
```

Comment:
(This comment is for the whole view) Is your view just a dummy test for now ? If it then it is ok, otherwise I would like to look a bit more like our current version of figma.

Reply:
Yes it's a dummy test, just to have a v1 skeleton of the model to be able to start with something when implmenting further features.